bundles/pom.xml

```diff
@@ -43,7 +43,34 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-scr-plugin</artifactId>
+        <version>1.24.0</version>
+        <configuration>
+          <supportedProjectTypes>
+            <supportedProjectType>eclipse-plugin</supportedProjectType>
+          </supportedProjectTypes>
+        </configuration>
+        <executions>
+          <execution>
+            <id>generate-scr-scrdescriptor</id>
+            <goals>
+              <goal>scr</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>      
     </plugins>
   </build>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.felix</groupId>
+      <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
```

Comment:
Not sure if I got this question right, but did you notice the "ds-" there?
 `org.apache.felix.scr.annotations` are "the old" felix ones, while `org.apache.felix.scr.ds-annotations` actually includes the official OSGi ones (see also [here](http://felix.apache.org/documentation/subprojects/apache-felix-maven-scr-plugin/apache-felix-maven-scr-plugin-use.html)). Or did I miss anything?